hellochain/tutorial/03-make.md

```diff
@@ -24,7 +24,6 @@ all: install
 
 install: go.sum
 	go install $(BUILD_FLAGS) ./cmd/hcd
-	go install $(BUILD_FLAGS) ./cmd/hccli
```

Comment:
I do believe this should stay here

Reply:
The order of the tutorial makes it such that `hccli` doesn't exist at the point the user `make install`s this make file - that's why I originally reordered the files, which is now in https://github.com/cosmos/sdk-tutorials/pull/185